Scenes/World.tscn

```diff
@@ -48,7 +48,7 @@ margin_left = 446.0
 margin_top = 269.0
 margin_right = 802.0
 margin_bottom = 309.0
-text = "Gold = "
```

Comment:
why is this change necessary wouldn't placeholder be better for understanding purpose